flarefly/fitter.py

```diff
@@ -664,6 +683,22 @@ def plot_mass_fit(self, **kwargs):
             plt.yscale('log')
             plt.ylim(min(total_func) * norm / 5, max(total_func) * norm * 5)
 
+        if show_extra_info:
+            chi2 = self.get_chi2()
+            ndf = self.get_ndf()
+            signal, signal_err = self.get_signal()
+            bkg, bkg_err = self.get_background()
+            s_over_b, s_over_b_err = self.get_signal_over_background()
+            significance, significance_err = self.get_significance()
+            text = AnchoredText(fr'$\chi^2 / \mathrm{{ndf}} =${chi2:.2f} / {ndf}''\n'
+                                fr'$S=${signal:.0f} $\pm$ {signal_err:.0f}''\n'
+                                fr'$B(3\sigma)=${bkg:.0f} $\pm$ {bkg_err:.0f}''\n'
+                                fr'$S/B(3\sigma)=${s_over_b:.2f} $\pm$ {s_over_b_err:.2f}''\n'
+                                fr'Signif.$(3\sigma)=${significance:.1f} $\pm$ {significance_err:.1f}',
+                                loc = 'center right',
+                                frameon=False)
+            axs.add_artist(text)
+
```

Comment:
This displays the additional information only for the the first signal, but if you have more it would be good to display all of them